### ISSUE TITLE
CY8CKIT_062_WIFI_BT: Failure to build the application for this target

### ISSUE THREAD

**Issue by hugueskamba** [ARMmbed/mbed-os-example-mbed-crypto#70]
### Description
Run the following command:
`mbed compile -t gcc_arm -m CY8CKIT_062_WIFI_BT`

Result:
```
[mbed] Working path "/path/to/mbed-os-example-mbed-crypto/getting-started" (program)
[mbed] WARNING: Could not find mbed program in current path "/path/to/mbed-os-example-mbed-crypto/getting-started".
       You can fix this by calling "mbed new ." in the root of your program.
---
Building project getting-started (CY8CKIT_062_WIFI_BT, GCC_ARM)
Scan: getting-started
Using RAM region application_ram in this build.
  Region application_ram: size 0x45800, offset 0x8002000
Compile [  0.1%]: mbed_tz_context.c
Compile [  0.2%]: main.cpp
[Fatal Error] main.cpp@17,10: psa/crypto.h: No such file or directory
[ERROR] ./main.cpp:17:10: fatal error: psa/crypto.h: No such file or directory
   17 | #include "psa/crypto.h"
      |          ^~~~~~~~~~~~~~
compilation terminated.

[mbed] ERROR: "/usr/local/opt/python/bin/python3.7" returned error.
       Code: 1
       Path: "/path/to/mbed-os-example-mbed-crypto/getting-started"
       Command: "/usr/local/opt/python/bin/python3.7 -u /path/to/mbed-os-example-mbed-crypto/getting-started/mbed-os/tools/make.py -t gcc_arm -m CY8CKIT_062_WIFI_BT --source . --build ./BUILD/CY8CKIT_062_WIFI_BT/GCC_ARM"
       Tip: You could retry the last command with "-v" flag for verbose output
---
```

### Issue request type

    [ ] Question
    [ ] Enhancement
    [√] Bug

**Comment by ciarmcom** [comment]
Thank you for raising this detailed GitHub issue. I am now notifying our internal issue triagers. 
*Internal Jira reference: https://jira.arm.com/browse/IOTCRYPT-1065*

**Comment by 0xc0170** [comment]
@Patater Was this target removed or how can we fix this?

cc @ARMmbed/team-cypress

**Comment by ciarmcom** [comment]
Thank you for raising this detailed GitHub issue. I am now notifying our internal issue triagers. 
*Internal Jira reference: https://jira.arm.com/browse/IOTOSM-2107*

**Comment by jesbun** [comment]
Exact same problem on NORDIC nRF52840-DK
```mbed compile -t GCC_ARM -m NRF52840_DK```

**Comment by rdvojmoc** [comment]
Same issue as stated above on NRF52840_DK target. Is there any fix? Tested with mbedOS v6.6.

**Comment by hugueskamba** [comment]
@jesbun and @rdvojmoc, I believe it's because these targets are not PSA targets.
The header file only exists for PSA targets:
```
$ find . -name "crypto.h"
./platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/inc/psa/crypto.h
./platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_V1_0/include/psa/crypto.h
./platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_TFM/TARGET_TFM_V1_1/include/psa/crypto.h
```
